### PR TITLE
move organization and project pages to /dashboard

### PR DESCRIPTION
**Pull request by avzz-19**

### CODE REVIEW

File: app/routes/authenticated/projects.ts [hunk 1]
```diff
@@ -1,4 +1,16 @@
...
+import RouterService from '@ember/routing/router-service';
+import Transition from '@ember/routing/transition';
+import { inject as service } from '@ember/service';
 
-export default class AuthenticatedProjectsRoute extends ScrollToTop(Route) {}
+export default class AuthenticatedProjectsRoute extends Route {
+  @service declare router: RouterService;
+
+  beforeModel(transition: Transition) {
+    const { params } = transition.to || {};
```
Comment (future-pirate-king) [inline app/routes/authenticated/projects.ts:10]: why do we require `params` here ?

File: tests/unit/controllers/authenticated/organization/users-test.js
Comment (future-pirate-king) [inline tests/unit/controllers/authenticated/organization/users-test.js:1]: this should be moved to `/authenticated/dashboard`

File: tests/unit/routes/authenticated/dashboard/organization-test.js [hunk 2]
```diff
@@ -1,13 +1,12 @@
+/* eslint-disable prettier/prettier */
```
Comment (future-pirate-king) [inline tests/unit/routes/authenticated/dashboard/organization-test.js:1]: remove this

File: tests/unit/routes/authenticated/dashboard/project-test.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+/* eslint-disable prettier/prettier */
```
Comment (future-pirate-king) [inline tests/unit/routes/authenticated/dashboard/project-test.js:1]: remove this

File: tests/unit/routes/authenticated/dashboard/project-test.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Route | authenticated/dashboard/project', function (hooks) {
+  setupTest(hooks);
+
+  test('it exists', function (assert) {
+    const route = this.owner.lookup('route:authenticated/dashboard/project');
+    var store = {
```
Comment (future-pirate-king) [inline tests/unit/routes/authenticated/dashboard/project-test.js:10]: use `const` and add new line above

File: tests/unit/routes/authenticated/dashboard/project-test.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+            id: 1,
+            type: 'project',
+            attributes: {
+              name: 'test',
+            },
+          },
+        ];
+      },
+    };
+    route.set('store', store);
```
Comment (future-pirate-king) [inline tests/unit/routes/authenticated/dashboard/project-test.js:23]: add new line above and below

File: tests/unit/routes/authenticated/dashboard/project/files-test.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
+/* eslint-disable prettier/prettier */
```
Comment (future-pirate-king) [inline tests/unit/routes/authenticated/dashboard/project/files-test.js:1]: remove this

File: tests/unit/routes/authenticated/dashboard/project/files-test.js [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+module(
+  'Unit | Route | authenticated/dashboard/project/files',
+  function (hooks) {
+    setupTest(hooks);
+
+    test('it exists', function (assert) {
+      const route = this.owner.lookup(
+        'route:authenticated/dashboard/project/files'
+      );
+      assert.ok(route);
```
Comment (future-pirate-king) [inline tests/unit/routes/authenticated/dashboard/project/files-test.js:14]: add new line above

File: tests/unit/routes/authenticated/organization/teams-test.js
Comment (future-pirate-king) [inline tests/unit/routes/authenticated/organization/teams-test.js:1]: this needs to be moved to `/authenticated/dashboard`

**Review by future-pirate-king — CHANGES_REQUESTED** [review]

File: tests/acceptance/deprecated-routes/project-redirect-test.js
Comment (future-pirate-king) [inline tests/acceptance/deprecated-routes/project-redirect-test.js:1]: can we add test for other projects related routes that is changed ?

**Review by future-pirate-king — CHANGES_REQUESTED** [review]

File: tests/unit/routes/authenticated/dashboard/organization/team-test.js
[the remark this answers is not in the document]
Comment (avzz-19) [reply tests/unit/routes/authenticated/dashboard/organization/team-test.js:1]: There is no route for this. It was changed to teams long back. Can I delete this test if not req?

[the remark this answers is not in the document]
Comment (future-pirate-king) [reply tests/unit/routes/authenticated/dashboard/organization/team-test.js:1]: yes

File: app/components/file/report-drawer/sbom-reports/index.ts [hunk 8]
```diff
@@ -75,7 +75,10 @@ export default class FileReportDrawerSbomReportsComponent extends Component<File
 
   @action async goToLatestFile() {
     this.args.closeDrawer();
-    this.router.transitionTo('authenticated.file', String(this.latestFileId));
+    this.router.transitionTo(
```
Comment (future-pirate-king) [inline app/components/file/report-drawer/sbom-reports/index.ts:78]: add new line above

File: app/components/marketplace/integration-list/index.ts [hunk 9]
```diff
@@ -11,19 +11,13 @@ export default class MarketplaceIntegrationListComponent extends Component {
...
         title: this.intl.t('jira'),
         description: this.intl.t('integrateAppknoxTo') + this.intl.t('jira'),
         logo: '../images/jira-icon.png',
-        link: 'authenticated.organization-settings',
-      },
-      {
-        title: this.intl.t('slack'),
-        description: this.intl.t('comingSoon'),
-        logo: '../images/slack-icon.png',
-        link: '',
```
Comment (future-pirate-king) [inline app/components/marketplace/integration-list/index.ts:26]: why is slack removed ?

File: tests/acceptance/deprecated-routes/projects-redirect-test.js [hunk 10]
```diff
@@ -0,0 +1,229 @@
...
+    assert.strictEqual(currentURL(), '/dashboard/project/1/files');
+
+    await visit('/project/1/settings');
+    assert.strictEqual(currentURL(), '/dashboard/project/1/settings');
+
+    await visit('/project/1/settings/analysis');
+    assert.strictEqual(currentURL(), '/dashboard/project/1/settings/analysis');
+
+    await visit(`/project/1/settings/dast-automation-scenario/${scenario.id}`);
+    assert.strictEqual(
```
Comment (future-pirate-king) [inline tests/acceptance/deprecated-routes/projects-redirect-test.js:224]: add new line above

**Review by future-pirate-king — CHANGES_REQUESTED** [review]

**Review by future-pirate-king — APPROVED** [review]

**Pull request merged**